src/grpc-plugin.ts

```diff
@@ -1,24 +1,29 @@
 import * as grpc from 'grpc';
+import * as protoLoader from '@grpc/proto-loader';
 import { Logger } from './logger';
 import { Browser } from './browser';
 
 const SERVER_ADDRESS = '127.0.0.1:50059';
 const RENDERER_PROTO_PATH = __dirname + '/../proto/renderer.proto';
-const GRPC_HEALTH_PROTO_PATH = __dirname + '/../proto/health.proto';
 
-export const RENDERER_PROTO = grpc.load(RENDERER_PROTO_PATH).models;
-export const GRPC_HEALTH_PROTO = grpc.load(GRPC_HEALTH_PROTO_PATH).grpc.health.v1;
+export const renderPackageDef = protoLoader.loadSync(RENDERER_PROTO_PATH, {
+  keepCase: true,
+  longs: String,
+  enums: String,
+  defaults: true,
+  oneofs: true,
+});
+
+export const rendererProtoDescriptor = grpc.loadPackageDefinition(renderPackageDef);
 
 export class GrpcPlugin {
   constructor(private log: Logger, private browser: Browser) {}
 
   start() {
     var server = new grpc.Server();
 
-    server.addService(GRPC_HEALTH_PROTO.Health.service, {
```

Comment:
Are you supposed to remove this health service?

Reply:
Upgraded grpc and got couldn't add the health service any longer. Read [here](https://grpc.io/docs/tutorials/basic/node/) and nothing mentioned the health stuff so tried to just remove it and seems to work :)

Reply:
Added it back. Apparently the hashicorp/go-plugin requires it: https://github.com/hashicorp/go-plugin/blob/master/docs/guide-plugin-write-non-go.md#3-add-the-grpc-health-checking-service

However, only seems to be called when client calls [Ping](https://github.com/hashicorp/go-plugin/blob/e8d22c780116115ae5624720c9af0c97afe4f551/protocol.go#L44) which Grafana currently doesn't do.

Reply:
Interesting, for running in k8s it would also be nice to have a /healthz http endpoint, right now I've been using / for readiness checks.